### PR TITLE
app.js: Project URLs changing on left/right navigation

### PR DESCRIPTION
**Pull request by gkrishnan724**
Fixes:https://github.com/coala/projects/issues/160

### CODE REVIEW

**Comment by gkrishnan724** [comment]
credits to @impmihai for the solution :)

File: resources/js/app.js [hunk 1]
```diff
@@ -4,18 +4,19 @@
 	app.config(['$routeProvider',
 		function($routeProvider) {
```
Comment (Adrianzatreanu) [inline resources/js/app.js]: hey could you remove the ``:`` from commit message? it only counts if you remove it for some reason :)

Comment (Adrianzatreanu) [inline resources/js/app.js]: also, your commit shortlog has no verb, making it hard to know what fix was done, could you rework it please? :)

File: resources/js/app.js [hunk 2]
```diff
@@ -4,18 +4,19 @@
...
 			}).
 			when('/mentors', {
 				template: '<mentors></mentors>'
 			}).
 			when('/faq', {
 				template: '<faq></faq>'
-			})
+			}).
+			otherwise({
+                redirectTo: '/projects'
```
Comment (hemangsk) [inline resources/js/app.js]: Indentation is a bit off

Comment (gkrishnan724) [inline resources/js/app.js]: :+1: I'll work on it

**Comment by hemangsk** [comment]
ack 1240f12

**Comment by hemangsk** [comment]
@rultor merge

**Comment by hemangsk** [comment]
ack ea77476

**Comment by hemangsk** [comment]
ack ea77476

**Comment by hemangsk** [comment]
@rultor merge

**Pull request merged**